utility/fisherman/module.go

```diff
@@ -0,0 +1,75 @@
+package fisherman
+
+import (
+	"github.com/pokt-network/pocket/logger"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/pokt-network/pocket/shared/modules/base_modules"
+)
+
+const (
+	FishermanModuleName = "fisherman"
+)
+
+type FishermanModule interface {
+	modules.Module
+
+	// TODO: add fisherman module functions here
+	FishermanUtility
+}
+
+// TODO_IN_THIS_COMMIT: exists to help with type assertions, drop once you've added your own functions
+type FishermanUtility interface {
```

Comment:
Let's move the `XXInterface` of each actor in `shared/utility_module.go` so its easy to understand and add functionality everywhere.